secure_uninstall/README.rst

```diff
@@ -0,0 +1,68 @@
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Secure Uninstall
+================
+
+Ask Master Password ('admin_passwd' key from config file)
+before to proceed to module uninstallation
+
+
+Configuration
+=============
+
+Check 'admin_passwd' is set in Odoo configuration file
+
+
+
+Usage
+=====
+
+Try to uninstall a module
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/149/server-tools
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/server-tools/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
```

Comment:
HI @sebalix, I don't know what it has changed. This is the exact copy of maintainer tools README template ?